External/SPMmodified/xASL/xASL_io_SaveNifti.m

```diff
@@ -16,6 +16,16 @@ function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, cha
 %                  masks, since it cannot contain a large data range
 %   bGZip          Gzip the result (OPTIONAL, DEFAULT 1)
 %   changeMat      New orientation matrix 4x4 (OPTIONAL, DEFAULT same as previous)
+%   bCopyOrigJson  Copy a JSON sidecar file (if existed for the original file) as sidecar of the new NIfTI (OPTIONAL, DEFAULT = false)
+%   JsonFields     Save additional JsonFields to a new JSON sidecar (or merged with the original Json content)
+%                  These fields overwrite fields with the same fieldnames in the original Json
+%                  (OPTIONAL, DEFAULT = struct() empty)
+% Options are:
+% 1. bSaveJson     = false                       -> we don't save a new JSON sidecar

```

Comment:
please remove line bSaveJson. It's not an input parameter anymore.

Reply:
Fixed in